### PR TITLE
Update to latest Cumulus

### PR DESCRIPTION
**Pull request by JoshOrndorff**
The main purpose of this PR is to update the workshop to the latest version of all software involved. Specifically, it:

* Uses the latest parachain template
* Uses the latest Polkadot (`rococo-branch`)
* Removes the workaround for obtaining genesis state
* Adds a `specs/snippets-to-copy.md` file to make it easier to update the specs in the future
* Removes the reference to `getsubstrate.io`, thus closing #9

It does **not**:
- Update the Polkadot JS screenshots.

### CODE REVIEW

File: 1-prep/1-compiling.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 This workshop covers the entire process of launching a relay chain, connecting parachains, transferring assets between
 chains, and developing your own parachain runtimes. Naturally, there will be some significant compiling if you intend to
-build everything yourself. (Compiling the Polkadot and test collator nodes can be avoided if you prefer to use the
-docker image.)
+build everything yourself. (Compiling the Polkadot and parachain template nodes can be avoided if you prefer to use the
+docker images.)
```
Comment (danforbes) [inline 1-prep/1-compiling.md:6]:
```suggestion
Docker images.)
```

File: 1-prep/1-compiling.md [hunk 2]
```diff
@@ -67,9 +63,8 @@ appropriate.
 The Substrate DevHub team maintains a
 [parachain template](https://github.com/substrate-developer-hub/substrate-parachain-template) (very similar to the
 [node template](https://github.com/substrate-developer-hub/substrate-node-template)) that we will use to launch our
-first parachains, and as a starting point for developing our own parachains. We will use this parachain template when
-learning to register parachains and do cross-chain asset transfers. Perform these steps in your typical workspace
-directory.
+first parachains and make cross-chain asset transfers, and as a starting point for developing our own parachains.
```
Comment (danforbes) [inline 1-prep/1-compiling.md:66]:
```suggestion
first parachains and make cross-chain asset transfers. Later, we will use it as a starting point for developing our own parachains.
```

File: 1-prep/2-chain-spec.md [hunk 3]
```diff
@@ -1,8 +1,7 @@
 # Obtaining a (Relay) Chain Specification
 
-You will need a chain specification for your relay chain network. There are three ways to obtain this spec, and you only
-need to do one. An important criteria to keep in mind is that you must always have one more relay chain validator than
-you have connected parachains. For example, if you want to connect two parachains, you need three validators in the
+You will need a chain specification for your relay chain network. You can use one that ships with this workshop, or create your own. An important criteria to keep in mind is that you must always have one more relay chain validator than
```
Comment (danforbes) [inline 1-prep/2-chain-spec.md:3]:
```suggestion
You will need a chain specification for your relay chain network. You can use one that ships with this workshop, or create your own. An important requirement to keep in mind is that you must always have one more relay chain validator than
```

**Review by danforbes — CHANGES_REQUESTED** [review]
Awesome changes :pray: Please run Prettier for line lengths.

File: specs/snippets-to-copy.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
+# Copyable snippets for chainspecs
```
Comment (riusricardo) [inline specs/snippets-to-copy.md:1]: Thanks for this :+1:

**Review by riusricardo — APPROVED** [review]
LGTM, just needs fixes in the lines length as Dan said.

**Review by danforbes — APPROVED** [review]
:pray:

**Comment by JoshOrndorff** [comment]
CI is unable to build the docker image because of https://github.com/paritytech/cumulus/issues/174

**Comment by JoshOrndorff** [comment]
Oh wow, the docker image built successfully on subsequent runs? That's good but also surprising.

**Comment by wheresaddie** [comment]
ran for me, no complaints :1st_place_medal:

**Pull request merged**